### DISCUSSION TITLE
Pretty link to applications

### DISCUSSION THREAD

**Discussion by eriverabe** [Budibase/budibase#3870] [Help] [answered]
Hello,

I think it would be a good idea to be able to configure pretty links for applications.
Actually a link to and app looks like `domain.com/app_78nas78nd8n8nasd8nas8d` and will be nice developer can configure some unique pretty app link like `domain.com/ticketingapplication`.

**Comment by mike12345567** [comment, marked as answer]
Hi @eriverabe there's some info about this in self host that can be found in this answer: https://github.com/Budibase/budibase/discussions/3811#discussioncomment-1835534

**Comment by eriverabe** [comment]
It worked! Thank you
It would be good if the platform uses these links automatically.
